src/cf-icons/src/cf-icons.less

```diff
@@ -44,24 +44,24 @@
 //
 // For example you can't do this:
 // .link-with-auto-icon {
-//   .@{cf-icon-prefix}();
+//   .@{cf-icon-prefix}(  );
 // }
 //
 // But you can do this:
 // .link-with-auto-icon {
-//   .@cf-icon();
+//   .@cf-icon(  );
 // }
 .cf-icon,
 .@{cf-icon-prefix} {
-  font-family: 'CFPB Minicons';
-  display: inline-block;
-  font-style: normal;
-  font-weight: normal;
-  line-height: 1;
-  // TODO: Determine whether this can be safely removed and remove it.
-  //       Font smooth/smoothing is non-standard:
-  //       https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
-  -webkit-font-smoothing: antialiased;
+    font-family: 'CFPB Minicons';
+    display: inline-block;
+    font-style: normal;
+    font-weight: normal;
+    line-height: 1;
+    // TODO: Determine whether this can be safely removed and remove it.
+    //       Font smooth/smoothing is non-standard:
+    //       https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
```

Comment:
Ok, I've done some quick testing.

1. Chrome on Windows is _not_ affected. I think because it uses DirectWrite for all text rendering on Windows.
   - Safari hasn't been supported on Windows since v5.
2. There are a lot of Mac users that don't have Retina Displays, or regularly plug into a non-Retina display.
3. You might want to check your assumptions about what looks good. Open up the sandbox in Chrome, scroll to the YouTube icon, inspect it, and flick the `-webkit-font-smoothing` on and off. Try both on your Thunderbolt Display and on the Retina Display.

Unless it's causing an issue, my vote at this time is to leave it.